PWGJE/Tasks/hadronPhotonCorrelation.cxx

```diff
@@ -39,6 +39,10 @@
 #include <Framework/InitContext.h>
 #include <Framework/runDataProcessing.h>
 
+#include "TClonesArray.h"
+#include "TLorentzVector.h"
+#include "TParticle.h"
+#include "TPythia8Decayer.h"
```

Comment:
Could you please change the include of ROOT and other outside includes to use <> instead of ""
In general for local includes (anything from O2Physics) in "" and everything else is considered outside and should use <>